keyboards/cannonkeys/satisfaction75/i2c_master.c

```diff
@@ -42,8 +42,8 @@ static const I2CConfig i2cconfig = {
   0
 };
 
-static i2c_status_t chibios_to_qmk(const msg_t* status) {
```

Comment:
Not following why this needed to be changed. If this did fix the oled, this should also be implemented in the arm version of i2c_master.

Reply:
All I know is I segfaulted when dereferencing status.
Rather than spend time debugging and root causing, I decided to to spend the extra 4 bytes - it's way cheaper.

Not sure if this is specific to just my use case or also to i2c_master.

Reply:
that's super odd, well if you are having the issue, I would suspect some arm i2c_master might also have the issue too, and since it is arm, as you said the 4 extra bytes wouldn't hurt. Mind updating that version as well?

Reply:
For now, I think I'm going to merge this. 

However, it is something worth following up on.